### PR TITLE
Add data disclaimer for pulumi ai

### PR DESCRIPTION
**Pull request by zchase**
## Description

This PR adds a disclaimer on the Pulumi AI project that data may be used to improve the results of Pulumi AI in the future

Fixes: https://github.com/pulumi/pulumi-ai/issues/25

## Checklist:

- [ ] I have reviewed the [style guide](https://github.com/pulumi/pulumi-hugo/blob/master/STYLE-GUIDE.md).
- [ ] If blogging, I have reviewed the [blogging guide](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md).
- [ ] I have manually confirmed that all new links work.
- [ ] I added aliases (i.e., redirects) for all filename changes.

### CODE REVIEW

**Review by AaronFriel — APPROVED** [review]
@susanev this looks good to me, can you take a look?

**Review by cnunciato — CHANGES_REQUESTED** [review]
Text is fine, but I'd love to take a crack at making this a little less visually dominating. Mind if I do that?

File: themes/default/content/ai/_index.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ welcome: |
 
 Pulumi AI is an experimental feature that lets you use natural-language prompts to generate Pulumi infrastructure-as-code programs in any language. This page is a web-based version of the open-source [Pulumi AI project](https://github.com/pulumi/pulumi-ai).
 
+{{% notes type="info" %}}
+The prompts you submit and the results may be used by Pulumi to improve the results of Pulumi AI in the future.
```
Comment (susanev) [inline themes/default/content/ai/_index.md:19]:
```suggestion
Submitted prompts and results may be used by Pulumi to improve Pulumi AI.
```

**Review by susanev — APPROVED** [review]

**Comment by susanev** [comment]
@cnunciato want to do a follow up pr instead? @anita-trimbur has new note styles we should use.

**Comment by cnunciato** [comment]
@zchase Let me know what you think -- I just tightened the words up a bit and removed the styling to keep the note from competing with the content above and below it.

**Review by cnunciato — APPROVED** [review]

**Comment by zchase** [comment]
@cnunciato Looks great!! Thank you!

**Pull request merged**